### DISCUSSION TITLE
ListTile Hover Effect

### DISCUSSION THREAD

**Discussion by burhansvural** [flet-dev/flet#2621] [General]
Can the button hover effect be applied when hovering over a ListTile?

**Comment by ndonkoHenri** [comment]
I dont seem to understand. Can you detail?

**Reply by burhansvural** [reply]
@ndonkoHenri When we hover the mouse over the button, the color of the button changes. Can I do the same when I hover over listTile?

**Reply by ndonkoHenri** [reply]
If the parent of your tile is a container for example, then you could make use of the `on_hover` handler to modify the colors. [Example](https://flet.dev/docs/controls/container#on_hover).
